### DISCUSSION TITLE
Google banned the use of SD on colab for free users

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#2086] [General]
Hey guys, I'm starting a new convo because I think it's important. Today, our suspicions about Google banning the use of SD for free users were confirmed. I found this message on Reddit:

"Google banned the usage of 'stable-diffusion-webui' on the free tier - no effect on the paid tier. There was a release yesterday/today of a1111 which changed the requirements, which will cause the Colab to not work. You'll need to wait for thelastben to update the notebook to use the new requirements."

I think this message should be added to Ben's Colab to prevent free users from complaining about it not working. It's a sad day, as I used to combine my paid accounts with free ones to get some extra hours each day. :(

**Comment by TheLastBen** [comment]
it still works for free users

**Reply by ClaraSanders** [reply]
So, that was just bullshit? I am so gullible to stay on the Internet.

**Reply by TheLastBen** [reply]
they just want to push people to pay more

**Reply by Headupdef** [reply]
I am having a ton of issues connecting now on free tier, keep getting this:
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 413, in <module>
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 292, in webui
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 173, in initialize
  File "/content/gdrive/.shortcut-targets-by-id/1eBhJesUCe0CTlDynt7ILJ5jF_r2_s8gq/sd/stable-diffusion-webui/modules/sd_models.py", line 100, in setup_model
  File "/content/gdrive/.shortcut-targets-by-id/1eBhJesUCe0CTlDynt7ILJ5jF_r2_s8gq/sd/stable-diffusion-webui/modules/sd_models.py", line 135, in list_models
  File "/content/gdrive/.shortcut-targets-by-id/1eBhJesUCe0CTlDynt7ILJ5jF_r2_s8gq/sd/stable-diffusion-webui/modules/sd_models.py", line 46, in init
  File "/content/gdrive/.shortcut-targets-by-id/1eBhJesUCe0CTlDynt7ILJ5jF_r2_s8gq/sd/stable-diffusion-webui/modules/sd_models.py", line 155, in model_hash
OSError: [Errno 107] Transport endpoint is not connected

**Reply by TheLastBen** [reply]
the issue is caused by  `.shortcut-targets-by-id/1eBhJesUCe0CTlDynt7ILJ5jF_r2_s8gq`

**Comment by Headupdef** [comment]
Is there any way to fix it?

**Comment by meunumerotim333** [comment]
Clearly this user from reddit is noob, he didn't see Ben fast updates all the problems. I'm using just fine. I also used models shortcuts from other Gdrive. No problems

**Reply by Headupdef** [reply]
can youy link me to the notebook then? every time i use the one from the main page it still does the same thing??

**Reply by meunumerotim333** [reply]
> can youy link me to the notebook then? every time i use the one from the main page it still does the same thing??

I've said that because the reddit user wrote a few days ago.. but just today I'm been disconnected everytime. This morning was working well. If I reconnect its working.. but just a few generations and them disconnects. Ben needs to tell us if there's some way to bypass this.

**Reply by Headupdef** [reply]
> > can youy link me to the notebook then? every time i use the one from the main page it still does the same thing??
> 
> I've said that because the reddit user wrote a few days ago.. but just today I'm been disconnected everytime. This morning was working well. If I reconnect its working.. but just a few generations and them disconnects. Ben needs to tell us if there's some way to bypass this.

I am having the exact same issue now.

**Comment by HaikenEdge** [comment]
Even though you get disconnected, you can still reconnect, and while you're connected, you can still generate images.

So, basically, it's more of a soft ban that forces users to babysit the notebook, rather than an outright ban that makes it unusable.

**Reply by meunumerotim333** [reply]
Yes, but each server are you using? Cause ngrok delays a lot to connect. It's annoying waiting so long to connect and them disconnects faster than connects.

**Reply by HaikenEdge** [reply]
I don't know what ngrok is, so I assume I'm just using a basic server.

**Reply by meunumerotim333** [reply]
Ngrok is this one:

![image](https://user-images.githubusercontent.com/121369033/236072285-f4b24e75-dbe6-4d2a-be20-6bacb543dd5f.png)

Its waaay more stable, I always get previews without bugs. But now with this shadowban isn't good anymore.

**Reply by HaikenEdge** [reply]
No, I mean, I knew it was an option, but I never messed around with it because I didn't see a reason to and I had no idea what it did.

**Reply by meunumerotim333** [reply]
I got a new weird situation.. I've reconnect a lot.. and now the images isn't been generating. I click on generate.. it shows "waiting".. and than nothing happens. Tried 4 times.. and same thing is happening. Maybe they are banning people that are forcing too much? Just a tip..

**Reply by Headupdef** [reply]
im telling you, it is blocked, they have blocked something in the script. I get all the way to the last cel and it just shuts down.

**Reply by meunumerotim333** [reply]
What about Colab Pro, will it work?
